doc/holochain_101/src/welcome.md

```diff
@@ -1,12 +1,24 @@
 # Welcome
 
 ## Status
-**This book is a work in progress. Articles which have content in them appear normally in the chapter navigation. Articles which are incomplete appear with a (E) next to their name, indicating it is an empty unwritten article.**
+**This documentation is a work in progress. Articles which have content in them appear normally in the chapter navigation. Articles which are incomplete appear with a (E) next to their name, indicating it is an empty unwritten article.**
+
+Hi there! You've discovered the comprehensive Holochain guidebook.
 
 Welcome, you've discovered the book on Holochain. It will guide you comprehensively through everything you need to know about the creation of fully peer-to-peer, secure, and decentralized applications built with Holochain. It also includes a section on contributing to, and extending, Holochain itself.
 
-Holochain is an open source software library that provides a comprehensive solution for businesses, communities, and any group of people to build and run applications which are hosted and validated by the "users" themselves, providing a superior level of agency and autonomy over heavy reliance on the so-called "cloud" and third parties. "Users" cease to merely "use" an application. By running it, they participate equally as nodes in the network, hosting and validating data, in addition to consuming it. Holochain provides a cross-platform framework for the development and execution of these applications. Applications can be developed utilizing any of the major operating systems in use, and run on any device with a 64-bit architecture. From desktops to laptops to Android (arm64) devices, a whole new set of opportunites open for the p2p applications you've been dreaming and scheming about.
+Holochain is an open source software library that provides a way for businesses, communities,
+and other groups to build and run applications which are hosted and validated by the "users" themselves.
+Doing so provides a superior level of agency and autonomy over heavy reliance on the so-called "cloud" and other third parties.
+
+These applications are known more widely as peer-to-peer, decentralized applications, or dApps. To distinguish between dApps built on Blockchains and those built on Holochain, we preemptively call the latter "hApps". A more detailed comparison between Blockchain dApps and Holochain hApps is available [here](https://medium.com/holochain/beyond-blockchain-simple-scalable-cryptocurrencies-1eb7aebac6ae).
```

Comment:
@ddd-mtl has started this document that we have worked on together in the core team: https://docs.google.com/document/d/1lFRQxrJqGI97A1S6WGsoL84qlVkP7HVjVF4cVA7zUF8/

We started to define hApps and dApps there as both something you can do with Holochain. Please have a look. We can still discuss these semantics. I think it makes sense to understand dApps not materially defined by blockchain but by decentralised. It would make sense to introduce a new term, hApps, for something that enables users to blend and combine UI components with Genomes (Holochain networks and micro-services..)

Just putting this here as FYI and a request for us all to get coherent on these terms.

Reply:
@lucksus big thanks! Please share any other docs like this, as it'll make it waaay easier for folks like @pdaoust and I to be effective here!

Reply:
@emalinus do you think that this merits making any changes to the text? I have a feeling best thing to do right now is merge, and set an intention and 'follow-ups' to refine the language/terms in the book

Reply:
Sure - there are other additions. Please do merge and of course we'll follow up and refine. The doc Nico sent will provide a lot of direction for later passes.

Reply:
added this to #343  mdbook QA checklist